src/mmw/js/src/analyze/views.js

```diff
@@ -313,6 +324,37 @@ var TableView = Marionette.CompositeView.extend({
     }
 });
 
+var LivestockTableRowView = Marionette.ItemView.extend({
+    tagName: 'tr',
+    template: livestockTableRowTmpl,
+    templateHelpers: function() {
+        return {
+            // Convert proportion to percentage for display.
+            proportion: (this.model.get('proportion') * 100),
```

Comment:
Is it possible to use [`toLocaleString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) here instead? Like we do in Program Analysis https://github.com/azavea/usace-program-analysis/blob/f9d891aa8af408998e44f7bd7709250ccc3e512e/src/usace_program_analysis/app/js/src/helpers/index.js#L69-L73

Although it should be noted that `toLocaleString` is not supported in Safari, so perhaps we're better off using this.